lib/generators/sufia/templates/config/legato.rb

```diff
@@ -0,0 +1,8 @@
+# If Google Analytics are enabled, pre-load the models which rely on
+# Legato::Model.extended(base) dynamic profile method generation. Non-production
+# environments wouldn't necessarily have this eager loaded by default.
+if Sufia.config.analytics
```

Comment:
If this doesn't affect production, can you add ` && Rails.env != "production"` or something equivalent?